public_html/class-mshots.php

```diff
@@ -23,6 +23,10 @@ class mShots {
 		const SCREEN_MAX_H = 3600;
 		const SCALE_FACTOR_VALUES = [1, 2];
 		const SCALE_FACTOR_DEFAULT = 1;
+		const FORMAT_TO_EXTENSION = [
```

Comment:
Could you make a block of the 3 new constants below the others.

So, `DEFAULT_FORMAT`, `ALLOWED_FORMATS`, `FORMAT_TO_EXTENSION`.